lib/matplotlib/tests/test_axes.py

```diff
@@ -9452,3 +9452,28 @@ def test_wrong_use_colorizer():
     for kwrd in kwrds:
         with pytest.raises(ValueError, match=match_str):
             fig.figimage(c, colorizer=cl, **kwrd)
+
+
+def test_bar_color_precedence():
+    # Test the precedence of 'color' and 'facecolor' in bar plots
+    fig, ax = plt.subplots()
+
+    # case 1: no color specified
+    bars = ax.bar([1, 2, 3], [4, 5, 6])
+    for bar in bars:
+        assert mcolors.same_color(bar.get_facecolor(), 'blue')
+
+    # case 2: Only 'color'
+    bars = ax.bar([1, 2, 3], [4, 5, 6], color='red')
```

Comment:
```suggestion
    bars = ax.bar([11, 12, 13], [4, 5, 6], color='red')
```
and similar for the following plots.

While drawing on top of each other technically works, it's harder to see what's going on when somebody tries to inspect the test result visually, e.g. in case we want to debug an error.